Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,59 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
+This is the default end-of-checkout script.
```

Comment:
I think it would be useful to add a link to the daytime checkout page.

Just a suggestion: maybe you can comment that sometimes we skip the calibrations, depending on day priorities. It may be important for new people, so they know that sometimes we don't do any calibration at all.